admin/class-gdpr-admin.php

```diff
@@ -1027,7 +1027,11 @@ public function add_consents_to_consents_column( $val, $column_name, $user_id )
 			} else {
 				$user_consents = get_user_meta( $user_id, 'gdpr_consents' );
 			}
-			return implode( ', ', $user_consents );
+			if ( $user_consents ) {
```

Comment:
I don't understand how this is supposed to work. Are you sure these are correct variables?
if `$user_consents` is set to anything other than empty value , you never reach the `else`, but when you do reach it, why would you need to `implode`?

Reply:
It's supposed to work the other way around. Updated.